omod/src/main/resources/config.xml

```diff
@@ -27,27 +27,31 @@
 			org.openmrs.module.uilibrary
 		</require_module>
 
-		<require_module version="1.0.2">
+		<require_module version="1.0.0">
 			org.openmrs.module.mchapp
 		</require_module>
         
         <require_module version="2.4">
             org.openmrs.module.appframework
         </require_module>
 		
-		<require_module version="1.5.1">
+		<require_module version="1.0.0">
 			org.openmrs.module.hospitalcore
 		</require_module>
 
+		<require_module version="17.0.4">
+			org.openmrs.module.kenyaemr
+		</require_module>
+
 		<require_module version="1.4">
 			org.openmrs.module.appui
 		</require_module>
-
+		<!--
 		<require_module version="1.0.1">
 			org.openmrs.module.referenceapplication
 		</require_module>
-
-		<require_module version="1.0.2">
+        -->
+		<require_module version="1.0.0">
```

Comment:
This version is suppossed to be the SNAPSHOT one? we don't have such a release that you are referring to